### ISSUE TITLE
Invalid escape sequence in template

### ISSUE THREAD

**Issue by ilionic** [coox/styled-jsx-css-loader#2]
Many thanks for the loader, saved many hours of trials and errors.
Getting this error:
```
Module build failed: SyntaxError: Invalid escape sequence in template
  1501 | .blockquote-reverse .small:after,
  1502 | blockquote.pull-right .small:after {
> 1503 |   content: '\00A0 \2014';
       |                    ^
  1504 | }
```
Also, related to escaping issue - icon variables are not rendering correctly, missing backslash when compiled ( i.e. https://github.com/rancher/icons/blob/master/variables.scss )

**Comment by coox** [comment]
Thank you for reporting this issue, @ilionic.

Proper escaping of CSS unicode sequences should now be fixed through 70d9abc, which is published on npm in version 0.3.0.

I’m closing this issue, please re-open if you encounter again.

**Comment by ilionic** [comment]
Works great, thank you! 👍
